plot-lib/plot/private/plot3d/point.rkt

```diff
@@ -51,7 +51,7 @@
                   #:sym [sym (point-sym)]
                   #:color [color (point-color)]
                   #:fill-color [fill-color 'auto]
-                  #:x-jitter [x-jitter (point-x-jitter)]
+                  #:x-jitter [#{x-jitter : Nonnegative-Real} (point-x-jitter)]
```

Comment:
Should the `Nonnegative-Real` type be applied to the `y-jitter` and `z-jitter` arguments below?

Reply:
That type annotation was unnecessary after I fixed other things; I've removed it.